migrations/20221016094040-update-add-permission-to-admin.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  async up(db) {
+    await db.collection("admins").updateMany({}, { $set: { permission: "" } });
+
+    await db
+      .collection("admins")
+      .updateOne({ email: "court.canva@gmail.com" }, { $set: { permission: "Super" } });
+  },
+
```

Comment:
if you add `@Prop() name: string;` in schema and set it `@IsNotEmpty()` in dto, please give admin `email: "court.canva@gmail.com"` a name here.

Reply:
thx changed the migration file;

module.exports = {
  async up(db) {
    await db
      .collection("admins")
      .updateOne(
        { email: "court.canva@gmail.com" },
        { $set: { permission: "Super", name: "Canva" } },
      );
  },

  async down(db) {
    await db
      .collection("admins")
      .updateOne({ email: "court.canva@gmail.com" }, { $unset: { permission: null, name: null } });
  },
};